### DISCUSSION TITLE
Wind Data

### DISCUSSION THREAD

**Discussion by Lewis2023** [albar965/littlenavmap#1090] [Q&A]
Hi
I set a Waypoint to the place of a Windbarb (NOAA ) and the Tooltip shows different wind directions for the WP and the NOAA Data.
What causes this difference?

**Comment by albar965** [comment]
Different altitude. The Yellowish wind barbs show wind at the flight plan altitude which can be lower than cruise for the climb or descent part. Tooltip shows this.

Wind at flight plan waypoint:
![grafik](https://github.com/albar965/littlenavmap/assets/15254956/f2a954d6-0e7b-4910-9ad8-a81ff69fa6c0)

Wind at selected altitude:
![grafik](https://github.com/albar965/littlenavmap/assets/15254956/e98910a7-4f5f-4b85-81f8-36a0ca8b17c1)
